Dockerfile.QA

```diff
@@ -304,6 +304,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
                               protobuf-compiler && \
     rm -rf /var/lib/apt/lists/*
 
+# CI/QA for memcheck requires valgrind
+RUN apt-get update && apt-get install -y valgrind
```

Comment:
Just merge with the above apt-get. You can add your comment to the existing one.